garnish/extensions.py

```diff
@@ -0,0 +1,105 @@
+"""
+Contains PyMOL extensions
+Is automatically imported when run from PyMOL
+Also automatically extends garnish to be called in PyMOL
+"""
+# Copyright 2019-2019 the garnish authors. See copying.md for legal info.
+
+from pymol import cmd
+from glob import glob
+import logging
+
+# local imports
+from . import Parser
+
+
+def bonds_from_system(selection, system, _self=cmd, quiet=1):
+    """
+    Take a System object and draw all the bonds in selection
+    :param system:
+    :param _self:
+    :param quiet:
+    :return:
+    """
+
+    for obj in _self.get_object_list(selection):
+        elastics_obj = obj + "_elastics"
+        _self.copy(elastics_obj, obj)
+        idx_shift = 0
+        for mol, n in system.topology:
+            for i in range(n):
+                draw_bonds(obj, mol.bonds, idx_shift)
+                draw_bonds(elastics_obj, mol.elastics, idx_shift)
+                idx_shift += len(mol)
+
+
+def draw_bonds(target, connections, idx_shift,
+               _self=cmd, quiet=1):
+    for a, b in connections:
+        a += idx_shift
+        b += idx_shift
+        _self.add_bond(target, a, b)
```

Comment:
Should you maybe still have the ability to draw bonds "slowly" if the pymol version does not have the `add_bond` function? Last time we did it like this:
```
for a, b in bonds:
    try:
        cmd.add_bond(obj, a, b)
    except AttributeError:
        cmd.bond(f"{obj} and ID {a}", f"{obj} and ID {b}")
```